dusty/changeset.py

```diff
@@ -0,0 +1,42 @@
+from .compiler.spec_assembler import get_repo_of_app_or_library, get_expanded_libs_specs
```

Comment:
This kind of sucks but I don't think it's going to create any circular dependencies in practice